pkg/kudoctl/util/repo/package.go

```diff
@@ -0,0 +1,308 @@
+package repo
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/kudobuilder/kudo/pkg/apis/kudo/v1alpha1"
+	"github.com/kudobuilder/kudo/pkg/bundle"
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	frameworkV1FileName     = "framework.yaml"
```

Comment:
difference between `V1` and `v1alpha1` and `v0.2.0` ?

Does `frameworkV1FileName` represent the new file structure from kep-9 and we call it V1?

Reply:
yeah I can drop the V1 if it's confusing. Basically I called the old one v0 and the kep-p v1